WordPress/src/main/res/layout/reader_cardview_post.xml

```diff
@@ -81,10 +81,35 @@
 
         </RelativeLayout>
 
-        <org.wordpress.android.widgets.WPNetworkImageView
-            android:id="@+id/image_featured"
-            style="@style/ReaderImageView.Featured.CardView"
-            android:layout_marginBottom="@dimen/margin_small" />
+        <FrameLayout
+            android:id="@+id/frame_photo"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <org.wordpress.android.widgets.WPNetworkImageView
+                android:id="@+id/image_featured"
+                style="@style/ReaderImageView.Featured.CardView"
+                android:layout_marginBottom="@dimen/margin_small"
+                tools:background="@color/grey_dark" />
+
+            <org.wordpress.android.widgets.WPTextView
+                android:id="@+id/text_photo_title"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="bottom|left"
+                android:layout_marginBottom="@dimen/margin_extra_large"
+                android:layout_marginLeft="@dimen/reader_card_content_padding"
+                android:layout_marginRight="@dimen/reader_card_content_padding"
+                android:ellipsize="end"
+                android:maxLines="1"
+                android:shadowColor="@color/grey_dark"
+                android:shadowDx="3.0"
```

Comment:
It's minor, but we could consider extracting these (`shadowDx`, `shadowDy`, `shadowRadius`) into `dimens.xml`.